### PR TITLE
Sweeping crumbs for grammatical fixes

### PR DESCRIPTION
**Pull request by bdlafleur**
## What is the change?

This PR sweeps the armi crumbs for grammatical fixes and tweaks.

---

## Checklist

- [x] This PR has only [one purpose or idea](https://terrapower.github.io/armi/developer/tooling.html#one-idea-one-pr).
- [x] [Tests](https://terrapower.github.io/armi/developer/tooling.html#test-it) have been added/updated to verify that the new/changed code works.

- [x] The code style follows [good practices](https://terrapower.github.io/armi/developer/standards_and_practices.html).
- [x] The commit message(s) follow [good practices](https://terrapower.github.io/armi/developer/tooling.html).

- [x] The [release notes](https://terrapower.github.io/armi/release/index.html) (location `doc/release/0.X.rst`) are up-to-date with any important changes.
- [x] The [documentation](https://terrapower.github.io/armi/developer/tooling.html#document-it) is still up-to-date in the `doc` folder.
- [x] No [requirements](https://terrapower.github.io/armi/developer/tooling.html#watch-for-requirements) were altered.
- [x] The dependencies are still up-to-date in `pyproject.toml`.

### CODE REVIEW

**Comment by keckler** [comment]
Reviewing now.

File: armi/reactor/tests/test_blocks.py [hunk 1]
```diff
@@ -1828,7 +1828,7 @@ def test_component_type(self):
         """
         Test that a hex block has the proper "hexagon" __name__.
 
-        .. test:: Ability to create hex shaped blocks
+        .. test:: User can create hex shaped blocks.
```
Comment (john-science) [inline armi/reactor/tests/test_blocks.py:1831]: I think this should be "Users can" or "A user can".

File: armi/reactor/tests/test_components.py [hunk 2]
```diff
@@ -197,7 +197,7 @@ def test_setNumberDensity(self):
     def test_setNumberDensities(self):
         """Test setting multiple number densities.
 
-        .. test:: Set Component number density
+        .. test:: User can set Component number densities.
```
Comment (john-science) [inline armi/reactor/tests/test_components.py:200]: I think this should be "Users can" or "A user can".

**Review by keckler — APPROVED** [review]
Who would have known that my PhD would one day allow me to become a copy editor?? :-D
Just kidding.

Looks great, thanks for all these quality changes.

**Review by john-science — APPROVED** [review]
Nice work!

**Pull request merged**